src/olympia/addons/tests/test_indexers.py

```diff
@@ -63,7 +63,7 @@ def expected_fields(cls, include_nullable=True):
         _indexed_translated_fields = ('name', 'description', 'summary')
         analyzer_fields = list(chain.from_iterable(
             [['%s_l10n_%s' % (field, lang) for lang, analyzer
-             in SEARCH_LANGUAGE_TO_ANALYZER.items()]
+              in SEARCH_LANGUAGE_TO_ANALYZER.items()]
```

Comment:
🐼

Reply:
Searching and replacing in VSCode seems to also do some automatic formatting which is not desired. 😞